iam/x509-identity-mgmt/js/config/default.conf

```diff
@@ -312,3 +312,98 @@ logger.file.max:string=7d
 logger.file.size:string=10m
 
 # ******************************************************************************
+
+# =======================================
+# kafka consumer - General configurations
+# =======================================
+
+# ---------------------------
+# X509IDMGMT_KAFKA_TOPIC_ACKS
+# ---------------------------
+kafka.topic.acks:integer=-1
+
+# ----------------------------------------
+# X509IDMGMT_KAFKA_TOPIC_AUTO_OFFSET_RESET
+# ----------------------------------------
+kafka.topic.auto.offset.reset:string=latest
```

Comment:
Maybe he could change to earliest, in case the service is started later, so it can receive device events created earlier.

Reply:
Well observed!
I will change this to make it as your suggestion. Good! :+1: